include/vcpkg/base/messages.h

```diff
@@ -1152,6 +1190,11 @@ namespace vcpkg
                     (),
                     "",
                     "vcpkg-ce ('configure environment') is experimental and may change at any time.");
+    DECLARE_MESSAGE(
+        VcpkgCompletion,
+        (msg::value, msg::path),
+        "'{value}' is the subject for completion. i.e. bash, zsh, etc.",
+        "vcpkg '{value}' completion is already imported to your \"{path}\" file.\nThe following entries were found: ");
```

Comment:
```suggestion
        "vcpkg {value} completion is already imported to your \"{path}\" file.\nThe following entries were found:");
```

I'm surprised that our validator let you get away with the trailing whitespace ....